.github/workflows/python-flake8.yml

```diff
@@ -3,7 +3,14 @@
 
 name: flake8
 
-on: [ push, pull_request ]
+on:
+  push:
+    branches:
+    - develop, stable
```

Comment:
from the yaml tutorial this should as list one item per line 
or you need brackets, e.g.  branches: [ develop, stable ]

see 
- https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started
- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

(I guess it searches now the whole string of the line)